### PR TITLE
Added Angular Dart Vs Code Extension

### PR DESCRIPTION
**Pull request by aadarshadhakalg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ This repository is started to celebrate [Hacktoberfest 2020](https://hacktoberfe
   - [Database Management Systems](#database-management-systems)
   - [Artificial Intelligence](#artificial-intelligence)
   - [Computer Graphics](#computer-graphics)
+
+- [Open Source Projects](#open-source-projects)
+  - [VS Code Extensions](#extensions)
```
Comment (saileshbro) [inline README.md]: I don't think we will be having more vs code extensions, can you change the VS code Extensions title to your project name? Keeping it under Open source projects

File: README.md [hunk 2]
```diff
@@ -134,7 +137,18 @@ This repository is started to celebrate [Hacktoberfest 2020](https://hacktoberfe
 
   [![Star this repo](https://img.shields.io/github/stars/saileshbro/double-pendulum-simulation)](https://github.com/saileshbro/double-pendulum-simulation)
 
+## Open Source Projects
+
+### VS Code Extensions
+
+- #### [Angular Dart](https://github.com/aadarshadhakalg/ADSnippets)
```
Comment (saileshbro) [inline README.md]: you can make this as ### heading

File: README.md [hunk 3]
```diff
@@ -134,7 +137,18 @@ This repository is started to celebrate [Hacktoberfest 2020](https://hacktoberfe
 
   [![Star this repo](https://img.shields.io/github/stars/saileshbro/double-pendulum-simulation)](https://github.com/saileshbro/double-pendulum-simulation)
 
+## Open Source Projects
+
+### VS Code Extensions
```
Comment (saileshbro) [inline README.md]: You can replace this with your project name

Comment (aadarshadhakalg) [reply README.md]: Sure

Comment (saileshbro) [reply README.md]: Thanks. and make sure you update the Table of contents too.

**Comment by aadarshadhakalg** [comment]
Done

**Review by saileshbro — APPROVED** [review]

**Pull request merged**